packages/client/lib/rpc/validation.ts

```diff
@@ -18,20 +18,19 @@ export function middleware(method: any, requiredParamsCount: number, validators:
         return reject(error)
       }
 
-      for (let i = 0; i < validators.length; i++) {
-        if (validators[i] !== undefined) {
-          for (let j = 0; j < validators[i].length; j++) {
+      for (let i = 0; i < params.length; i++) {
```

Comment:
would like to understand the reason for validators to params switch :slightly_smiling_face: